Makefile

```diff
@@ -0,0 +1,8 @@
+build:
+	@dune build @install
+
+clean:
+	@dune clean
+
+test-all:
+	SIHL_ENV=test dune runtest --force --no-buffer test
```

Comment:
@mikonieminen We can remove `SIHL_ENV` imo

Reply:
oops

Reply:
fixed